bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_10/A5_10_10.java

```diff
@@ -18,11 +18,40 @@
 /**
  *
  * @author Daniel Weber - Initial contribution
+ * 
+ *         From A5_10_10 up to A5_10_14 temperature is given as a 8Bit value (range: 0..250!).
+ *         Therefore higher values mean higher temperatures.
+ *         Temperature range 0..40.
```

Comment:
I can't make suggestions to existing code, but can you reformat this to:
```java
/**
 * From A5_10_10 up to A5_10_14 temperature is given as a 8Bit value (range: 0..250!).
 * Therefore higher values mean higher temperatures.
 * Temperature range 0..40.
 *
 * @author Daniel Weber - Initial contribution
 */
```

This will ensure it will be correctly shown in Javadoc and IDE's.